src/Riok.Mapperly/ISourceProductionContext.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Riok.Mapperly;
+
+public interface ISourceProductionContext
+{
+    public void AddSource(string hintName, Microsoft.CodeAnalysis.Text.SourceText sourceText);
+    public void AddSource(string hintName, string source);
```

Comment:
these three methods are totally comes from `SourceProductionContext`, maybe we should keep them

Reply:
Since this method is completely unused I‘d remove it for now. It can be readded if it is needed. Otherwise we need to maintain it although it is not used.